### PR TITLE
fix (bbb-web): Null pointer exception when parent meeting does not exist

### PR DESCRIPTION
**Pull request by paultrudel**
### What does this PR do?

Performs a check when creating a breakout room through the API to see if the parent meeting exists.

### Motivation

Currently no check is done to see if the meeting corresponding to the provided `parentMeetingID` actually exists which can lead to an error.

### CODE REVIEW

**Comment by antobinary** [comment]
Follow up from https://github.com/bigbluebutton/bigbluebutton/pull/18975

File: bigbluebutton-web/grails-app/controllers/org/bigbluebutton/web/controllers/ApiController.groovy [hunk 1]
```diff
@@ -127,9 +127,15 @@ class ApiController {
       return
     }
 
-    if(params.isBreakout == "true" && !params.parentMeetingID) {
-      invalid("parentMeetingIDMissing", "No parent meeting ID was provided for the breakout room")
-      return
+    if(params.isBreakout == "true") {
```
Comment (gustavotrott) [inline bigbluebutton-web/grails-app/controllers/org/bigbluebutton/web/controllers/ApiController.groovy:130]:
```suggestion
    Boolean isBreakoutRoom = false
    if (!StringUtils.isEmpty(params.isBreakout)) {
      isBreakoutRoom = Boolean.parseBoolean(params.isBreakout)
    }

    if(isBreakoutRoom) {
```

File: bigbluebutton-web/grails-app/controllers/org/bigbluebutton/web/controllers/ApiController.groovy [hunk 2]
```diff
@@ -127,9 +127,15 @@ class ApiController {
...
+      if(!params.parentMeetingID) {
```
Comment (gustavotrott) [inline bigbluebutton-web/grails-app/controllers/org/bigbluebutton/web/controllers/ApiController.groovy:131]:
```suggestion
      if(StringUtils.isEmpty(params.parentMeetingID)) {
```

**Review by gustavotrott — CHANGES_REQUESTED** [review]
Let's follow the pattern used to read other params.

**Review by gustavotrott — APPROVED** [review]

**Pull request merged**